content/automate/ManageIQ/Transformation/Infrastructure/VM/vmwarews.class/__methods__/setmigrated.rb

```diff
@@ -28,8 +28,9 @@ def main
                 exit MIQ_OK
```

Comment:
Currently, the code exits right at the beginning of `main`. If we want the tag assignment to be active, the `exit MIQ_OK` statement should be placed after `source_vm.tag_with(...)`.

Reply:
To take this a step further it would be better to remove the `exit MIQ_OK` line completely and change this method to only do the tagging.

The other logic should either be removed or at the least moved into a new method in this script.

@fdupont-redhat Are there plans to use this logic soon?  If not let's remove the unused code.

cc @mkanoor